prometheus/operators/aws-environment.yml

```diff
@@ -0,0 +1,220 @@
+---
+
+# Deploy to the Support Network in AWS
+- type: replace
+  path: /instance_groups/name=alertmanager/networks/name=default/name?
+  value: Support
+
+- type: replace
+  path: /instance_groups/name=prometheus/networks/name=default/name?
+  value: Support
+
+- type: replace
+  path: /instance_groups/name=database/networks/name=default/name?
+  value: Support
+
+- type: replace
+  path: /instance_groups/name=grafana/networks/name=default/name?
+  value: Support
+
+- type: replace
+  path: /instance_groups/name=nginx/networks/name=default/name?
+  value: Support
+
+- type: replace
+  path: /instance_groups/name=firehose/networks/name=default/name?
+  value: Support
+
+# Update storage volume sizes for persistent services
+- type: replace
+  path: /instance_groups/name=alertmanager/persistent_disk_type?
+  value: 10GB
+
+- type: replace
+  path: /instance_groups/name=prometheus/persistent_disk_type?
+  value: 10GB
+
+- type: replace
+  path: /instance_groups/name=database/persistent_disk_type?
+  value: 10GB
+
+- type: replace
+  path: /instance_groups/name=grafana/persistent_disk_type?
+  value: 10GB
+
+# add monitoring vm_extension to instances (for security groups)
+- type: replace
+  path: /instance_groups/name=alertmanager/vm_extensions?
+  value: [monitoring]
+
+- type: replace
+  path: /instance_groups/name=prometheus/vm_extensions?
+  value: [monitoring]
+
+- type: replace
+  path: /instance_groups/name=database/vm_extensions?
+  value: [monitoring]
+
+- type: replace
+  path: /instance_groups/name=grafana/vm_extensions?
+  value: [monitoring]
+
+- type: replace
+  path: /instance_groups/name=nginx/vm_extensions?
+  value: [monitoring]
+
+- type: replace
+  path: /instance_groups/name=firehose/vm_extensions?
+  value: [monitoring]
+
+# deploy small instances
+- type: replace
+  path: /instance_groups/name=alertmanager/vm_type?
+  value: t2.small
```

Comment:
`cf-deployment` recently changed from using names like `t2.small` to `small` or `minimal`. We added a bunch of VM types yesterday to the cloud-config to reflect. It might be more desirable to use these if want to be portable across providers?